ports/mimxrt/Makefile

```diff
@@ -156,6 +159,7 @@ SRC_C += \
 	modmimxrt.c \
 	moduos.c \
 	mphalport.c \
+	fatfs_port.c \
```

Comment:
please put this in sorted order, I'd suggest after `dma_channel.c`

Reply:
done. Ordered alphabetically now.